dev/tools/lib/roll_dev.dart

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 The Chromium Authors. All rights reserved.
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
please don't change copyright dates

Reply:
It wasn't incorrect. This script is a derived work of a work created in 2017.

Reply:
Ahh.  Ok, then.  I'll revert it.